inspec.yml

```diff
@@ -0,0 +1,377 @@
+name: cms-ars-5.0-microsoft-windows-server-2016-stig-overlay
+title: cms-ars-5.0-microsoft-windows-server-2016-stig-overlay
+maintainer: CMS SAF Team
+copyright: 
+copyright_email: saf@cms.hhs.gov
+license: Apache-2.0
+summary: CMS ARS 5.0 Overlay InSpec Validation Profile for Windows Server 2016 STIG
+version: 1.7.0
+inspec_version: ">= 4.0"
+
+depends:
+  - name: microsoft-windows-server-2016-stig-baseline
+    url: https://github.com/mitre/microsoft-windows-server-2016-stig-baseline/archive/parameterize.tar.gz
```

Comment:
Note to change the branch name back after testing and before merging

Reply:
Thanks for the reminder